client/tests/test_client.py

```diff
@@ -48,7 +48,7 @@ def test_fail_to_add_client_password(self, client0):
         short_password = '123'
```

Comment:
if the reason for the failure is a short password, at least mention it in the test name.

Reply:
also how short is too short? maybe parameterized this, or use the longest-yet-short option.

Reply:
Done!